### ISSUE TITLE
Badresource bug

### ISSUE THREAD

**Issue by wamuM** [wamuM/playing-cards#1]
When refreshing the client after a while sometimes the server crashes with this error (commit sha: 3ab8b5f)
```
error: Uncaught (in promise) BadResource: Bad resource ID
                        await requestEvent.respondWith(new Response("404 Not Found",{status:404}));
                        ^
    at async Object.respondWith (deno:ext/http/01_http.js:240:11)
    at async file:///C:/Users/wamu_M/Documents/Codes/PlayingCards/server/server.js:72:25
```

**Comment by wamuM** [comment]
Maybe related, sometimes the server freezes but the client doesn't give up (I'm assuming part of the connection like the tcp handshake or the http, happens and then something breaks) and if you refresh the client it gets the files but the server crashes with this BadResource error in the same line

**Comment by wamuM** [comment]
This bug seems to have been fixed as of commit 9cc9f50 
It was caused because the server would try to answer an already answered request
